src/app/apps/lti/views.py

```diff
@@ -89,6 +90,19 @@ def render_to_response(self, context=None, **response_kwargs):
             context = {}
 
         context["app_data"] = json.dumps(self.app_data)
+
+        # Pass App's API and Warren's API base URLs to the frontend.
+        # If not specified in the .env, frontend should declare their values.
+        context.update(
+            {
+                key: os.getenv(key, "")
+                for key in [
+                    "WARREN_FRONTEND_API_ROOT_URL",
+                    "WARREN_FRONTEND_APP_ROOT_URL",
+                ]
+            }
+        )
```

Comment:
A better way to handle this is to add those two variables to the settings (that can be overridden by the environment) and use them here as follow:

```suggestion
        # To put on top (if not already loaded)
        from django.conf import settings
        
        context.update(
            {
                "WARREN_API_ROOT_URL": settings.API_ROOT_URL,
                "WARREN_APP_ROOT_URL": settings.APP_ROOT_URL,
            }
        )
```